src/utils/find-files.ts

```diff
@@ -11,6 +11,8 @@ export const isSchema = (filePath: string): boolean =>
     R.split('/')
   )(filePath)
 
+export const isJson = (filePath: string): boolean => filePath.endsWith('.json')
+
 export const findFiles = (
```

Comment:
is this function still used anywhere after this PR is merged?

Reply:
Just in `findConfigFiles()` below